examples/cat.go

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) 2019, 2020 Aleksa Sarai <cyphar@cyphar.com>
+// Copyright (C) 2019, 2020 SUSE LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Examples should be licensed under the LGPL-3.0-or-later (**only** the binding code is Apache-2.0) See the other examples for the license text to copy.

Also, given that you wrote this (and the binding) feel free to add your own `// Copyright (C) 2020 ...` line with your name for this file and the other binding file.